park-dashboard/src/Components/StandardParkInfo/StandardInfo.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { LoadParkData } from "../../LoadParkData";
+import "./StandardParkInfo.css";
+
+const StandardInfo = ({ parks, parkCode }) => {
+  const parkData = LoadParkData(parks);
+  const stateWiseNationalParks = parkData.filter(
+    (park) => park.designation === "National Park" && park.parkCode === parkCode
```

Comment:
To include all park information, including generic parks, it is advisable to remove the "National Park" filter. Instead, I suggest using the following code: const stateWiseNationalParks = parkData.filter((park) => park.parkCode === parkCode);. This will filter parks based on the specific park code.

Please refer to the image where the park is not designated as a National Park
<img width="1344" alt="image" src="https://github.com/rutujapadgilwar/Park-Dashboard/assets/106363277/35569d63-214d-4963-9edd-0aad48bfd09b">

Reply:
Added the error message if no information available.